src/cpp/ext/filters/census/client_filter.cc

```diff
@@ -68,53 +75,42 @@ constexpr uint32_t
     OpenCensusCallTracer::OpenCensusCallAttemptTracer::kMaxTagsLen;
 
 //
-// OpenCensusClientChannelData
+// OpenCensusClientFilter
 //
 
-grpc_error_handle OpenCensusClientChannelData::Init(
-    grpc_channel_element* /*elem*/, grpc_channel_element_args* args) {
+const grpc_channel_filter OpenCensusClientFilter::kFilter =
+    grpc_core::MakePromiseBasedFilter<
+        OpenCensusClientFilter, grpc_core::FilterEndpoint::kClient,
+        grpc_core::kFilterExaminesServerInitialMetadata>("opencensus_client");
```

Comment:
I don't see where this examines server initial metadata?

Reply:
Thanks! Yeah, I think I misread it as client initial metadata. This makes more sense!